app/sidekiq/hca/ezr_submission_job.rb

```diff
@@ -12,11 +12,9 @@ class EzrSubmissionJob
     sidekiq_options retry: 14
 
     sidekiq_retries_exhausted do |msg, _e|
-      log_submission_failure(decrypt_form(msg['args'][0]))
-    end
-
-    def self.log_submission_failure(parsed_form)
-      Form1010Ezr::Service.new(nil).log_submission_failure(parsed_form)
+      Form1010Ezr::Service.new(nil).log_exhausted_submission_failure(
```

Comment:
Hmm. Any reason we would want to keep this in the service versus just inlining it here (whether in another method or just in this method?). Especially since this is the only place we'll be calling it.

Reply:
Good point. I'll move it and refactor the tests.